tv_apps/smart-system/js/crash_reporter.js

```diff
@@ -26,9 +25,19 @@ var CrashReporter = (function() {
 
   // This function should only ever be called once.
   function showDialog(crashID, isChrome) {
-    var title = isChrome ? _('crash-dialog-os2') :
-      _('crash-dialog-app', { name: crashedAppName });
-    document.getElementById('crash-dialog-title').textContent = title;
+    if(isChrome) {
```

Comment:
For such kind of iff statement, we should write as the following to prevent some jshint warning:
```
var title = isChrome ? {
                  id: 'crash-dialog-os2',
                  args: null
                } : {
                  id: 'crash-dialog-app',
                  args: null
                };
```